doc/source/f2py/moddata_session.dat

```diff
@@ -1,11 +1,12 @@
 >>> import moddata
->>> print moddata.mod.__doc__
-i - 'i'-scalar
-x - 'i'-array(4)
-a - 'f'-array(2,3)
-foo - Function signature:
-  foo()
+>>> print(moddata.mod.__doc__)
+'i'-scalar
```

Comment:
The names of module data should be restored.

Reply:
Same as above, depends on #15325